internal/versionsapi/cli/list.go

```diff
@@ -137,10 +138,6 @@ func listPatchVersions(ctx context.Context, client *verclient.Client, ref string
 			return nil, fmt.Errorf("listing patch versions: %w", err)
 		}
 
-		if len(list.Versions) == 0 {
-			return nil, fmt.Errorf("no versions found")
-		}
-
```

Comment:
This function is also called within the remove command, did you consider that? Look good nevertheless, but would we great if you could check this, too, if you didn't already. :smile_cat:

Reply:
I'm quite sure this still works correctly but I'll run a quick test to be sure!